draft-ietf-trans-gossip.md

```diff
@@ -401,30 +409,43 @@ with the following content:
     representation of the binary SCT data as defined in
     {{RFC-6962-BIS-09}} Section 3.3.
 
-The 'x509\_chain' element MUST contain at least the leaf certificate
-and SHOULD contain the full chain to a root accepted by all of the
-logs in the set of logs issuing all the SCTs in the 'sct\_data'
-element.
+The 'x509\_chain' element MUST contain the full chain from the leaf certificate 
+to a trust anchor (either pre-loaded or locally added) accepted by all of the 
+logs in the set of logs issuing all the SCTs in the 'sct\_data' element.
+
```

Comment:
I think this is a change from what we talked about last call... We can talk about it more this call.